packages/components/src/Banner/Banner.css

```diff
@@ -1,27 +1,63 @@
 :root {
   --flash--inset: var(--space-base);
+  --flash--inset--small: var(--space-small);
 }
 
 .flash {
   display: flex;
-  margin-bottom: var(--space-base);
-  padding: var(--flash--inset);
-  border-top: var(--border-thick) solid var(--color-grey--light);
-  border-bottom-right-radius: var(--space-smaller);
-  border-bottom-left-radius: var(--space-smaller);
+  box-shadow: var(--shadow-base);
+  padding: var(--flash--inset--small) var(--flash--inset);
   color: var(--color-black);
   background-color: var(--color-grey--lightest);
-  align-items: center;
+  align-items: start;
```

Comment:
This give a warning in `docz`. Let's change to `flex-start`

![image](https://user-images.githubusercontent.com/46460840/103798327-c5978700-5006-11eb-9b70-107e34c409cd.png)